bindings/src/lib.rs

```diff
@@ -54,6 +54,13 @@ pub fn check_dfa(test: &str, input: &str) -> Result<(), String> {
         };
         Err(e)
     } else {
+        if test == "assignment-4-4" {
+            let mut minimized_dfa: Dfa = solution_dfa.clone();
+            minimized_dfa.minimize();
+            if proposed_dfa.states().len() != minimized_dfa.states().len() {
+                return Err(format!("The provided DFA is not minimal"));
```

Comment:
Maybe we should explicitly tell the students in this case that the DFA accepts the correct language, but is not minimal.